manifests/install.pp

```diff
@@ -9,6 +9,12 @@
     before => Class['qpid::tools'],
   }
 
+  if $qpid::auth {
+    package { 'cyrus-sasl-plain':
+      ensure => present,
+    }
```

Comment:
It may be better to avoid potential duplicate declarations by using [ensure_packages](https://forge.puppet.com/modules/puppetlabs/stdlib/reference#ensure_packages):
```suggestion
    ensure_packages(['cyrus-sasl-plain'])
```

I can imagine it may be declared elsewhere too.